widget-src/components/log/LogEditing.tsx

```diff
@@ -166,7 +238,21 @@ export const ChangeLogEditing = ({
                       },
                     },
                   });
+
                   setUpdatedDate(Date.now());
+
+                  saveToVersionHistory(
+                    nameText,
+                    saveType,
+                    saveChange,
+                    saveLinks,
+                    showName,
+                    showTypes,
+                    showVersion,
+                    version,
+                  )
+                    .then(val => console.log('Version History Saved', val))
```

Comment:
nit: `val` is showing as `undefined` in the console every time so it probably isn't worth passing through.